components/sliders/SliderProjects/index.tsx

```diff
@@ -0,0 +1,144 @@
+'use client';
+import React, { useState } from 'react';
+import Slider from 'react-slick';
+
+import ArrowLeft from '@/components/sliders/ArrowLeft';
+import ArrowRight from '@/components/sliders/ArrowRight';
+import useFetch from '@/hooks/useFetch';
+import { Department, DepartmentProject } from '@/types/departments';
+
+import DepartmentCardPicture from './DepartmentCardPicture';
+import DepartmentCardText from './DepartmentCardText';
+import DepartmentMoblieCard from './DepartmentMobileCard';
+import SkeletonDepartmentCard from './SkeletonDepartmentCard';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import '@/components/sliders/styles.css';
+
+const SliderProjects = ({ depId }: { depId: number }) => {
```

Comment:
SliderProjects must accept `projects` prop. Slider will be used in department and club pages, so we shouldn't delegate extra fetching in this component